content/en/blog/_posts/2024-05-01-cri-streaming-explained.md

```diff
@@ -0,0 +1,341 @@
+---
+layout: blog
+title: "Container Runtime Interface streaming explained"
+date: 2024-05-01
+slug: cri-streaming-explained
+---
+
+**Author:** Sascha Grunert
+
+The Kubernetes [Container Runtime Interface (CRI)](docs/concepts/architecture/cri)
```

Comment:
The blog content contains a handful of links that need fixing.

```suggestion
The Kubernetes [Container Runtime Interface (CRI)](/docs/concepts/architecture/cri)
```

Reply:
Ah yes, fixed them.